README.md

```diff
@@ -124,9 +126,9 @@ your CSS goes here</code></pre>
 </div>
 ```
 
-This is base starting point for all CSS examples. Your next step is to fill in the example element. For `border-raius` it makes sense to have a simple div element with a solid background color. For this example then, the already present `example-element` div will do, however we do need to give it some basic styling, and perhaps add the text "Style Me".
+This is the base starting point for all CSS examples. Your next step is to fill in the example element. For `border-radius` it makes sense to have a simple `&lt;div&gt;` element with a solid background color. The already present `example-element` `&lt;div&gt;` will do, however we do need to give it some basic styling, and perhaps add the text "Style Me".
```

Comment:
Here you do not need to escape the < and >. Because it is inside back ticks, it will be processed as code. If escaped as above, the output will be `&lt;div&gt;` instead of `<div>`